ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -366,16 +403,19 @@ export default defineComponent({
       context.emit('cancel')
     }
 
-    watch(() => localState.owner.organizationName, async (val: string) => {
-      if (val.length >= 3) {
-        const result: SearchResponseI = await searchBusiness(val)
-        if (!result.error) {
-          console.log(result.searchResults)
-        } else {
-          console.log(result.error)
+    watch(
+      () => localState.owner.organizationName,
+      async (val: string) => {
+        if (val.length >= 3) {
+          const result: SearchResponseI = await searchBusiness(val)
+          if (!result.error) {
+            console.log(result.searchResults)
+          } else {
+            console.log(result.error)
+          }
```

Comment:
I prefer the previous formatting but your call, just a preference.

Reply:
Yea, me too, but it's just the way prettier formats it. I tried other formatters and changed a few options, but it messes up other things.

Reply:
Fair enough, only concern is reformatting done unintentionally in any file you work in. 
As long as you keep an eye on it 👍